### PR TITLE
maintainers: add pantheon team

### PR DESCRIPTION
**Pull request by davidak**
###### Motivation for this change

Needed for https://github.com/NixOS/nixpkgs/issues/121827

Related to https://github.com/NixOS/nixpkgs/pull/121780 and https://github.com/NixOS/nixpkgs/issues/121769

Potential additions:

- [x] Add more members
- [x] Add team to maintainers of packages (when https://github.com/NixOS/nixpkgs/pull/121780 is merged, so we don't have a merge conflict)

but it could also be merged in the current state. let's wait some more

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by bobby285271** [comment]
@davidak Will like to join this if the team is still looking for people :-)

```nix
bobby285271 = {
  email = "rjl931189261@126.com";
  github = "bobby285271";
  githubId = 20080233;
  name = "Bobby Rong";
};
```

P.s.: `pkgs/desktops/pantheon/default.nix` will also need update:

https://github.com/NixOS/nixpkgs/blob/4be510e3adfbe630e32e8d92feebaa603270e68f/pkgs/desktops/pantheon/default.nix#L26

**Comment by davidak** [comment]
@bobby285271 can you review and  check if something **not** belongs to pantheon? i think it is good if we get notified about vala changes because it could break pantheon or apps. does that make sense?

we can add the team to other components in the other pr or later if something is missing

**Review by bobby285271 — APPROVED** [review]

File: nixos/modules/services/desktops/geoclue2.nix [hunk 1]
```diff
@@ -266,5 +266,8 @@ in
       } // mapAttrs' appConfigToINICompatible cfg.appConfig);
   };
 
-  meta.maintainers = with lib.maintainers; [ ];
+  meta = with lib; {
+    maintainers = with maintainers; [ ] ++ teams.pantheon.members;
```
[the remark this answers is not in the document]
Comment (davidak) [reply nixos/modules/services/desktops/geoclue2.nix:270]: @NixOS/freedesktop what do you think?

File: nixos/modules/services/desktops/tumbler.nix [hunk 2]
```diff
@@ -18,8 +18,8 @@ in
       "")
   ];
 
-  meta = {
-    maintainers = with maintainers; [ ];
```
[the remark this answers is not in the document]
Comment (davidak) [reply nixos/modules/services/desktops/tumbler.nix:22]: is it used by pantheon or only xfce?

[the remark this answers is not in the document]
Comment (bobby285271) [reply nixos/modules/services/desktops/tumbler.nix:22]: I think tumbler is [part of the XFCE desktop](https://docs.xfce.org/xfce/tumbler/start), but the service is enable by default in both XFCE and Pantheon installations.

File: nixos/modules/services/desktops/geoclue2.nix (hunk 1 above)
[the remark this answers is not in the document]
Comment (bobby285271) [reply nixos/modules/services/desktops/geoclue2.nix:270]: [Just found this](https://github.com/NixOS/nixpkgs/pull/121780/files#r626486577), maybe will be just fine for us to pick this if both jtojnar and 7c6f434c didn't.

File: nixos/modules/services/desktops/geoclue2.nix [hunk 3]
```diff
@@ -266,5 +266,8 @@ in
...
+  };
+
 }
```
Comment (SuperSandro2000) [inline nixos/modules/services/desktops/geoclue2.nix:273]:
```suggestion
  };
}
```

**Comment by davidak** [comment]
@SuperSandro2000 i squashed your change in. can we merge now?

**Comment by bobby285271** [comment]
Almost forget this, we probably need to do something similar to https://github.com/NixOS/nixpkgs/pull/83975 (switch from pantheon.maintainers to lib.teams.pantheon).

**Comment by davidak** [comment]
Most cases already use `teams.pantheon.members` similar to `teams.gnome.members.`

**Pull request merged**